sql/core/src/main/scala/org/apache/spark/sql/execution/SparkQl.scala

```diff
@@ -249,8 +249,9 @@ private[sql] class SparkQl(conf: ParserConf = SimpleParserConf()) extends Cataly
 
         DropFunction(dbName, funcName, ifExists.isDefined, temp.isDefined)(node.source)
 
-      case Token("TOK_ALTERTABLE", alterTableArgs) =>
-        AlterTableCommandParser.parse(node)
+      case Token(fieldName, alterTableArgs)
```

Comment:
You can also use: `Token("TOK_ALTERTABLE" | "TOK_ALTERVIEW", alterTableArgs) =>`